scripts/tuple_tree_generator/tuple_tree_generator/templates/tuple_tree_gen.py.tpl

```diff
@@ -0,0 +1,254 @@
+###- This template file is distributed under the MIT License. See LICENSE.md for details. -###
```

Comment:
Outline everything that does not strictly need to be generated.

Reply:
That piece is not included in the generated file (`### marks a template comment ###`), do you want to remove it anyway?

Reply:
No, I mean, all the classes that have nothing generated. Outline them.